### PR TITLE
Add rubocop config and correct violations

### PR DESCRIPTION
**Pull request by dkniffin**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
+
+AllCops:
+  NewCops: disable
+  TargetRubyVersion: 2.3
```
Comment (oboxodo) [inline .rubocop.yml:14]: @dkniffin 2.3? That old?

Comment (dkniffin) [reply .rubocop.yml:14]: Oh, whoops. I just copied what I had here, but without the `>=` https://github.com/Quimbee/importmap-package-manager/blob/main/importmap-package-manager.gemspec#L11 🤦‍♂️ I'll update this

Comment (dkniffin) [reply .rubocop.yml:14]: Hmm, actually those two values should match, so I did that right. But 2.3 might be older than we want to support. It looks like [importmap-rails supports 2.7+](https://github.com/rails/importmap-rails/blob/main/importmap-rails.gemspec#L17). I think I'll go with that same value, unless you can think of a reason not to.

Comment (dkniffin) [reply .rubocop.yml:14]: PR for that here: https://github.com/Quimbee/importmap-package-manager/pull/3

Comment (oboxodo) [reply .rubocop.yml:14]: I think following same supported version as `importmap-rails` is a great idea.

**Pull request merged**